### DISCUSSION TITLE
Kata for Javascript:  Converting a Python variable name to Camel Case for Javascript.

### DISCUSSION THREAD

**Discussion by JeffGreenlee42** [codewars/codewars.com#3130] [General]
I'm just posting a question here.  I just finished the Kata exercise "Convert string to Camel Case".    I failed the test because it produced this error:  
"toCamelCase('The-Stealth-Warrior') did not return correct value: expected 'the-stealth-warrior' to equal 'TheStealthWarrior'" 

But my understanding of the Javascript convention of camel case is that the first word must start with a lowercase letter?    The exception is if  one were converting a string to Pascal case.    So I added an if statement to my algorithm to treat the first word differently than the rest. 

Am I misinformed?

**Comment by KayleighWasTaken** [comment]
Yeah that kata is misnamed, nothing we can do about it now though as far as I know.

**Comment by JeffGreenlee42** [comment]
Ok ,  got it!  (Sigh!). My solution is rather more inelegant than some of
the other solutions.  Didn’t think to use regular expression!  🤨

Jeff Greenlee

"Music is a higher revelation than all wisdom and philosophy. Music is the
electrical soil in which the spirit lives, thinks and invents."
Ludwig Van Beethoven
